src/main/java/core/basesyntax/service/impl/TransactionParserServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.TransactionParserService;
+import core.basesyntax.transaction.FruitTransaction;
+import core.basesyntax.transaction.Operation;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TransactionParserServiceImpl implements TransactionParserService {
+    private static final String COMA_SEPARATOR = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> getTransactions(List<String> linesFormFile) {
+        linesFormFile = skipFirstLine(linesFormFile);
+        List<FruitTransaction> fruitTransactionList = new ArrayList<>();
+        for (String line : linesFormFile) {
+            FruitTransaction fruitTransaction = parseTransaction(line);
+            fruitTransactionList.add(fruitTransaction);
+        }
+        return fruitTransactionList;
```

Comment:
Can you rewrite it using only Stream API? :)
```suggestion
        return linesFromFile.stream()
            ... (magic here);
```